### ISSUE TITLE
Repeater field enhancement

### ISSUE THREAD

**Issue by kokers** [elementor/elementor#2862]
1. after clicking "add new item" in repeater field, it would be great if the newly added field was collapsed + first text input field focused. This would speed up a process of adding new items.

2. repeat should act as a toggle not accordion + there should be an option to toggle/collapse all items with one click. Currently when clicking some item we need to scroll up because element position changed. It's very jumpy. 

3. In the preview it would be great if the repeater field had an option "+ add row / - remove row" directly in preview. This would benefit editing flow a lot. The remove row should be with confirmation prompt. 

4. Option to move in preview editing repeater items between two sections would be great. Like for example I have team widget with heading + items and use two of them on the page and when I need to move some items between those section, I need to remove and add it, when I could simply drag and drop it to other section.

**Comment by bainternet** [comment]
@kokers 

Thanks for this suggestion.

1.  It was already requested and we are thinking about.

2. This is something we have talked about but the majority of use-cases would need to scroll too much and we passed on it.

3. Nice Idea, I can see how the editing flow will benefit from it. so we are considering.

4. This would be great but sadly, it is not really possible because each widget is an entity on its own and can't be aware of other widgets.

Again thanks for your suggestions.
